Vizzuality/landgriffon
# Includes updated excel template and new indicators names

This PR encompasses an updated version of the data template available for download through the application. Additionally, we have ensured the alignment of indicator names and metadata with the Langriffon methodology v2.0.

### Testing instructions

- Ensure that the base data ingestion works with the new indicators and units csv files.  
- Please ensure that the foundational data ingestion process is functioning correctly, allowing users to successfully download the new template from the data interface and subsequently upload it after it has been filled out.





## Checklist before merging
